### DISCUSSION TITLE
[Question]Mobile app Responsive UI

### DISCUSSION THREAD

**Discussion by KarimovichS** [flet-dev/flet#4472] [Q&A]
### Question

How can the elements of a page be made responsive?

The documentation mentions ResponsiveRow, but is it sufficient on its own?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by SIRGPrice** [comment]
You have to play with adding containers surrounding the elements and setting expand = True, also you can set expand = True on the element so it ocuppy all the space. Or you can use native responsive elements like responsive row.
